pnet-idp-client-saml2/src/main/java/at/porscheinformatik/idp/saml2/Saml2Utils.java

```diff
@@ -12,7 +14,11 @@
 
 public class Saml2Utils
 {
-    private static final String RELAY_STATE_PARAM = "RelayState";
+    static final String RELAY_STATE_PARAM = "RelayState";
+    static final String AUTO_GENERATED_RELAY_STATE_FORMAT = "AutoGenerated%s|%s";
```

Comment:
Sorry for this. I forgot to update my comment. Please use a "-" instead of the "|". | has a special meaning in a Regex and will break the code in `getRelayState`

Reply:
No problem, I change `|` to `-`